library/socket/basicsocket/recv_nonblock_spec.rb

```diff
@@ -113,6 +113,7 @@
       end
 
       ruby_version_is ""..."3.3" do
+        quarantine! do # May fail with "IO::EAGAINWaitReadable: Resource temporarily unavailable - recvfrom(2) would block" error
```

Comment:
I would say retry on `IO::WaitReadable` for `client.recv_nonblock(10)`, it just hasn't seen the closed socket yet.

Reply:
Done in https://github.com/ruby/ruby/pull/12710

Reply:
Have no clue why these new specs fail on Windows - they raise `Errno::EINVAL: Invalid argument - recvmsg(2)`

https://github.com/ruby/spec/pull/1240/commits/ccc72c7cf5e832ba1b02691e16a5b2f50e9d6662